### DISCUSSION TITLE
你们的域名都是怎么搞得？未成年.jpg

### DISCUSSION THREAD

**Discussion by CreeperIsASpy** [Meloong-Git/PCL#5224] [提问] [answered]
如题：

阿里云、腾讯云、……        ->     🚫 未成年人禁止，且还要上传身份证照片，不能得到信任啊
> 我妈担心我把她卖了.jpg

Cloudflare等国外网站        ->     🤕 梯子……你搞得了，你牛。

**Comment by zkitefly** [comment]
eu.org

**Reply by Deep-Dark-Forest** [reply]
实际上大部分免费二级域名国内都或多或少会有污染……
而且原作者也说了不能国外网站

**Reply by Ad-closeNN** [reply]
1. https://nic.us.kg
2. https://www.cloudns.net

**Reply by WorldHim** [reply]
> 实际上大部分免费二级域名国内都或多或少会有污染……
> 而且原作者也说了不能国外网站

cf 和 eu.org 都没墙

**Comment by Hill23333** [comment]
未成年人不禁止吧？
~如果真的不行试试[阿里云国际站](https://www.alibabacloud.com)（需要google账号或非+86手机号）~
国际站也需要身份证或户口本

**Comment by hejiehao** [comment]
有身份证和身份证照片就可以了

**Reply by Hill23333** [reply]
> 还要上传身份证照片，不能得到信任啊

**Reply by hejiehao** [reply]
什么叫不能得到信任

**Reply by Hill23333** [reply]
我也不清楚 (￣ω￣;)

**Reply by Deep-Dark-Forest** [reply]
> 什么叫不能得到信任

应该是父母不把身份证给他。怕隐私泄露

**Comment by WorldHim** [comment]
cf 没墙…

**Reply by Deep-Dark-Forest** [reply]
> CloudFlare 在中国大陆地区的连通性一直不太好
近期有报告称部分运营商网络对 CloudFlare 产生 SNI 阻断，导致有的人打不开了……

#5014

**Reply by hejiehao** [reply]
https://cloudflare.com 没事

**Comment by xxtg666** [comment]
我之前用过 namesilo，那里可以随便填身份信息，你可以试试
（原来现在阿里云禁止未成年认证了吗？我之前搞好了到现在一直能用）

其实梯子这个东西……你都能上 github 了，肯定也能找到不少的。

二编：查了下文档，阿里云不是满 14 岁就可以么？如果你没满 14 的话等等也不急的。

**Reply by MoYuan-CN** [reply]
阿里云我 14 岁的时候实名的是可以过的

**Reply by LinQingYuu** [reply]
>原来现在阿里云禁止未成年认证了吗

认证倒没禁，Author 指的应该是域名 ICP 备案

**Reply by CreeperIsASpy** [reply]
> 等等也不急的

 两年后我快中考了，我怕我没时间……
~~完了暴露年龄了~~

**Reply by Hill23333** [reply]
很奇怪，在阿里云文档里我的地区支持16岁以上备案，但是我备案失败。原因是管局要求18...

**Reply by LinQingYuu** [reply]
> 很奇怪，在阿里云文档里我的地区支持16岁以上备案，但是我备案失败。原因是管局要求18...

可能是过期文档（

**Comment by CreeperIsASpy** [comment]
> 你都能上 github 了

属地: 上海
上海目前已经对github没有墙了，只要有公网ipv6就可以访问……
但是，这个时灵时不灵，需要watt toolkit加持才行。
~~据说是忘记封ipv6了~~

**Reply by CreeperIsASpy** [reply]
据 @Ad-closeNN 的描述，阿里云应该是要14+，但我……~~自己看QQ身份卡~~
~~我在这里就不提广告猫猫伪造身份证的伟大事迹了~~

eu.org嘛…，我要的是顶级！~~我有钱，有钱，包有钱的！~~

**Reply by CreeperIsASpy** [reply]
关于CF， 我没有国外信用卡，年龄不够.png

> 应该是要14+，但我……~~自己看QQ身份卡~~

**Reply by hejiehao** [reply]
试试这个？
https://www.rainyun.com/

**Reply by Hill23333** [reply]
那你这就比较无解了 freenom现在也不给了

**Reply by Hill23333** [reply]
> 试试这个？ https://www.rainyun.com/

雨云也需要身份证啊

**Reply by hejiehao** [reply]
> > 试试这个？ https://www.rainyun.com/
> 
> 雨云也需要身份证啊

好像也没说要 14

**Reply by LinQingYuu** [reply]
> 据 @Ad-closeNN 的描述，阿里云应该是要14+，但我……~自己看QQ身份卡~ ~我在这里就不提广告猫猫伪造身份证的伟大事迹了~
> 
> eu.org嘛…，我要的是顶级！~我有钱，有钱，包有钱的！~

how？
你可以考虑找别人帮你买域名，但是你大概是没办法应对 ICP 备案的（

**Reply by WorldHim** [reply]
> 据 @Ad-closeNN 的描述，阿里云应该是要14+，但我……~~自己看QQ身份卡~~
> ~~我在这里就不提广告猫猫伪造身份证的伟大事迹了~~
> 
> eu.org嘛…，我要的是顶级！~~我有钱，有钱，包有钱的！~~

cf 认 eu.org 作为顶级

**Reply by Deep-Dark-Forest** [reply]
> > 据 @Ad-closeNN 的描述，阿里云应该是要14+，但我……~~自己看QQ身份卡~~
> > ~~我在这里就不提广告猫猫伪造身份证的伟大事迹了~~
> > 
> > eu.org嘛…，我要的是顶级！~~我有钱，有钱，包有钱的！~~
> 
> cf 认 eu.org 作为顶级

有没有一种可能他说的是要求域名看起来是顶级，而不是 cf 认它为顶级

**Comment by Light-Beacon** [comment, marked as answer]
域名购买应该不用成年，只有ICP备案有18+硬性指标
很介意实名的话可以去国外服务商买，绝大部分都不要求实名（除了一些特定地方域名）

**Reply by CreeperIsASpy** [reply]
确实，概括的挺全面的

**Comment by BarbTurnip437** [comment]
可以用gitgub
